### PR TITLE
Fix lessjs

### PR DESCRIPTION
**Pull request by biyeun**
@benrudolph 
related: https://github.com/dimagi/commcarehq-prelogin/pull/32

### CODE REVIEW

File: corehq/apps/style/management/commands/fix_less_imports_collectstatic.py [hunk 1]
```diff
@@ -23,11 +21,25 @@ def handle(self, *args, **options):
...
-                    print "[FIXING] bad less import in %s" % less_file
-                    content = content.replace(NON_COLLECTED_PATH, COLLECTED_PATH)
-                else:
-                    content = None
+                if content is not None:
+                    B3_REGEX = r"@\{b3-import-[a-z]+-[a-z]+\}"
+                    p = re.search(B3_REGEX, content)
+                    if p is not None:
+                        path_def = p.group(0)
+                        path_file = path_def.split('-')[-2]
```
Comment (benrudolph) [inline corehq/apps/style/management/commands/fix_less_imports_collectstatic.py]: maybe just a comment here to say what this is doing? it looks like you're determining whether it's a mixin or variables. or just add grouping to the regex

File: corehq/apps/style/management/commands/fix_less_imports_collectstatic.py [hunk 2]
```diff
@@ -23,11 +21,25 @@ def handle(self, *args, **options):
...
+                        new_path = settings.LESS_B3_PATHS.get(path_file)
+                        print ("[CORRECTING] %(f_name)s path in %(less_file)s: "
```
Comment (benrudolph) [inline corehq/apps/style/management/commands/fix_less_imports_collectstatic.py]: maybe move this to under the `new_path` if statement? not sure if it's a bad thing if it gets this far and `new_path` is `None`

**Comment by benrudolph** [comment]
:+1: from me. just some minor comments

File: corehq/apps/style/management/commands/fix_less_imports_collectstatic.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/style/management/commands/fix_less_imports_collectstatic.py]: also if this is just a one-off command it's probably not necessary

**Comment by biyeun** [comment]
@benrudolph responded

**Comment by benrudolph** [comment]
cool, will merge after tests

**Pull request merged**